<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="lbim1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Pump 1 (GPIO21)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbim2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Pump 2 (GPIO20)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbim3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Pump 3 (GPIO26)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ingredient#1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ingredient#2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ingredient#3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ingredient#4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Back to main menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbim4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Pump 4 (GPIO16)</string>
   </property>
  </widget>
  <widget class="QLabel" name="limbm5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
